beacon_node/http_api/src/lib.rs

```diff
@@ -3627,6 +3631,7 @@ pub fn serve<T: BeaconChainTypes>(
                 .uor(get_beacon_block_attestations)
                 .uor(get_beacon_blinded_block)
                 .uor(get_beacon_block_root)
+                .uor(get_blobs)
```

Comment:
sorry to be pedantic but why in this order? it's a bit of breaking the list of `beacon_block_` ones. Can we move it a bit down?

Reply:
yeah ok maybe it makes sense. Ignore my pedantic comment if you want